account/account.go

```diff
@@ -69,15 +73,25 @@ func newKeystoreAccount(keystorePath, password, opAccount string) (*keystoreAcco
 		return nil, err
 	}
 
-	return &keystoreAccount{ks, account}, nil
+	return &keystoreAccount{ks, address, account, fullNode}, nil
 }
 
-func (k *keystoreAccount) PayBidTx(ctx context.Context, fullNode node.FullNode, receiver common.Address, amount *big.Int) ([]byte, error) {
-	nonce, err := fetchNonce(ctx, fullNode, k.account.Address)
+func (k *keystoreAccount) PayBidTx(ctx context.Context, receiver common.Address, amount *big.Int) ([]byte, error) {
+	// take pay bid tx as block tag
+	if amount == nil {
+		amount = big.NewInt(0)
+	}
+
+	balance, nonce, err := prev(ctx, k.fullNode, k.address)
```

Comment:
can we have a metrics on the sentry node about the balance of the account, so that we can monitor and alert?

Reply:
added